docs/spectrum/index.rst

```diff
@@ -10,110 +10,116 @@ Spectrum estimation and modeling (`gammapy.spectrum`)
 in publications usually the reference [Piron2001]_ is used.
 A standard reference for the unfolding method is [Albert2007]_.
 
-.. _spectrum_getting_started:
 
-Getting Started
-===============
+Spectral Fitting
+================
 
-Spectral fitting within Gammapy is most easily performed with the ``gammapy-spectrum`` command line tool. 
+.. _spectrum_command_line_tool:
 
-The spectral fitting command-line tool makes use of the data management functionality in Gammapy. In order to download an example dataset from the `gammapy-extra <https://github.com/gammapy/gammapy-extra>`__ repository and set up an example `gammapy.obs.DataManager` please follow the instructions in :ref:`obs_dm`. The following step assume you have this example data set. If you already have a data set, please modify the steps below accordingly.
+Command line tool
+-----------------
+
+Spectral fitting within Gammapy is most easily performed with the ``gammapy-spectrum`` command line tool.
+The example below shows how to use ``gammapy-spectrum`` by specifying analysis
+options in a YAML config file. It assumes you have the `gammapy-extra <https://github.com/gammapy/gammapy-extra>`__
+repository available.
 
-This is an example config file (YAML format) to be used with the ``gammapy-spectrum`` command line tool.
 
 .. include:: ./analysis_example.yaml
     :code: yaml
 
 
-Copy it to for example ``crab_config.yaml`` and run
+Copy the above config file to your machine, to e.g. ``crab_config.yaml`` and run
 
 .. code-block:: bash
 
    gammapy-spectrum crab_config.yaml
 
-.. _energy_handling_gammapy:
 
-Energy handling in Gammapy
-==========================
+Underlying classes
+------------------
 
-Basics
-------
+The spectral fitting procedure is a two step process. Each of the two steps is represented by one class.
 
-Most objects in Astronomy require an energy axis, e.g. counts spectra or
-effective area tables. In general, this axis can be defined in two ways.
+* The `~gammapy.spectrum.SpectrumAnalysis` class converts the data from the
+  Fermi-LAT format (add link) into the OGIP format
+* The `~gammapy.spectrum.SpectralFit` class calls Sherpa in order to fit a
+  model to the data
 
-* As an array of energy values. E.g. the Fermi-LAT diffuse flux is given at
-  certain energies and those are stored in an ENERGY FITS table extension.
-  In Gammalib this is represented by GEnergy.
-* As an array of energy bin edges. This is usually stored in EBOUNDS tables,
-  e.g. for Fermi-LAT counts cubes. In Gammalib this is represented by GEbounds.
+Creating OGIP data
+^^^^^^^^^^^^^^^^^^
 
-In Gammapy both the use cases are handled by two seperate classes: 
-`gammapy.utils.energy.Energy` for energy values and
-`gammapy.utils.energy.EnergyBounds` for energy bin edges
+The following examples creates the 4 OGIP files that are needed for a spectral analysis
 
-Energy
-------
+* `PHA file <https://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/spectra/ogip_92_007/node5.html>`__
```

Comment:
We should have a small section describing PHA, ARF, RMF in the spec repo.
It can be very short and reference the OGIP specs and I think it's worth having there even if we don't have anything extra.

But probably there are some details worth mentioning, like header keywords we use or the fact that we don't have channels, but reco energy?

@joleroi - Do you agree it's useful to have a very short page on ARF, RMF, PHA there instead of referencing OGIP directly or describing things in Gammapy? Can you add it?